web_readonly_bypass/static/src/js/readonly_bypass.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Allow to bypass readonly fi the value is changed
+ */
+
+openerp.web_readonly_bypass = function(instance) {
+
+    var QWeb = instance.web.qweb, _t = instance.web._t;
+    /**
+     * ignore readonly: place options['readonly_fields'] into the data
+     * if nothing is specified into the context
+     *
+     * create mode: remove read-only keys having a 'false' value
+     *
+     * @param boolean mode: True case of create, false case of write
+     * @param {Object} context->filter_out_readonly
+     * @param {Object} data field values to possibly be updated
+     * @param {Object} options Dictionary that can contain the following keys:
+     *   - readonly_fields: Values from readonly fields to merge into the data object
+     */
+    function ignore_readonly(data, options, mode, context){
+        if (options){
+            if ('readonly_fields' in options && options['readonly_fields'] &&
+                    !('filter_out_readonly' in context && context['filter_out_readonly'] == true )) {
```

Comment:
@JonathanNEMRY , @pedrobaeza 

To be accurate, my problem was that, when I quickly tested to define context on the field like this on the view:
```xml
<field name="my_readonly_field" context="{'filter_out_readonly': True}" />
```
or on the action windows. 

The javascript ``context`` parameter does'nt contain the ``filter_out_readonly`` properties :-/

It needs more  investigations ;-)